src/main/java/org/hibernate/infra/bot/config/RepositoryConfig.java

```diff
@@ -15,6 +15,7 @@ public static class JiraConfig {
 		private Optional<Pattern> issueKeyPattern = Optional.empty();
 
 		private Optional<Boolean> insertLinksInPullRequests = Optional.empty();
+		private Optional<Integer> issueLinksLimit = Optional.of( 3 );
```

Comment:
If there's a default, no need to use `Optional`... or am I missing something?

Reply:
Yes of course, just copy-pasted from other config properties and forgot to change that. Thanks!